leaf-ui/js/onFunctions.js

```diff
@@ -693,18 +640,13 @@ $('#btn-clear-cycle').on('click',function(){
 $('#btn-clear-analysis').on('click', function() {
     // reset to default analysisRequest while preserving userAssignmentsList
     resetToDefault();
-    // clear analysis sidebar
-    clearAnalysisConfigSidebar();
     // remove all configs from analysisMap
     analysisMap.clear();
-	// add back first default analysis config
-    addFirstAnalysisConfig();
     // reset graph to initial values
     revertNodeValuesToInitial();
 });
 
 $('#btn-clear-results').on('click', function() {
-    clearResults();
```

Comment:
Maybe remove refreshAnalysisUI() and leave a TODO in this clear result/clear config functions as well to go back and re-implement